docs/tutorials/solve-higher-order-binary-optimization-problems-with-q-ctrls-optimization-solver.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
     "{/* cspell:ignore lambdify ILOG histtype stepfilled */}\n",
     "\n",
-    "# Solve higher-order binary optimization problems with Q-CTRL's Optimization Solver"
+    "# Higher-order binary optimization problems with Q-CTRL's Optimization Solver"
```

Comment:
Can I remove the word "problems" from this title @miamico ? Otherwise it sounds like the tutorial is about the solver being problematic, rather than solving problems with it :-)

Reply:
Sounds good!